From ad2f6068f3acec8a790ec94f8069dbabf28c960a Mon Sep 17 00:00:00 2001
From: Swee Yee Fonn <swee.yee.fonn@intel.com>
Date: Thu, 31 Dec 2020 11:10:23 +0800
Subject: [PATCH] Update start_flash_usb.sh for keybox provisioning

Update start_flash_usb.sh to remove
RPMB_DATA file also for guest image
in disk partition case.

Change-Id: I2323d95ade46ab83072b11cbbc1f8ca0a8e65836
Tracked-On: OAM-95598
Signed-off-by: Swee Yee Fonn <swee.yee.fonn@intel.com>
---
 scripts/start_flash_usb.sh | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/scripts/start_flash_usb.sh b/scripts/start_flash_usb.sh
index 82b960f..b2da6ac 100755
--- a/scripts/start_flash_usb.sh
+++ b/scripts/start_flash_usb.sh
@@ -78,6 +78,10 @@ parse_arg "$@" || exit -1
 if [[ $FLASH_PHY_DISK == true ]]; then
   dd if=/dev/zero of=$GUEST_IMAGE bs=32M count=8
   mkfs.ext4 $GUEST_IMAGE
+  if [ -f RPMB_DATA ]
+  then
+    rm RPMB_DATA
+  fi
 else
   if [ -f $GUEST_IMAGE ]; then
     echo -n "$GUEST_IMAGE already exsited, Do you want to flash new one(y/N):"
--
2.17.1

